one_fm/utils.py

```diff
@@ -2593,7 +2594,6 @@ def get_mandatory_fields(doctype, doc_name):
 	if not employee_fields:
 		for link_field in meta.get_link_fields():
 			if link_field.options == 'Employee':
-				employee_fields.append(link_field.fieldname)
```

Comment:
Why it is removed

Reply:
Well, this was added even when no employee field exist. And it later gave me error saying the field was Nonetype.

Reply:
I added it back, with some other changes.

Reply:
If no employee fields in mandatory fields then we fetch employee linked fields and add to the employee_fields list to set the employee data.

Reply:
There are cases in which the employee fields are not mandatory by doctype definition